engine/admin/logging/awslogs.md

```diff
@@ -63,6 +63,18 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+### tag
+
+Specify `tag` as an alternative to the `awslogs-stream` option. `tag` interprets template markup (e.g., `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` `docker.{{.ID}}`). 
+See the [tag option documentation](log_tags.md) for details on all supported template substitutions.
+
+When both `awslogs-stream` and `tag` are specified, the value supplied for `awslogs-stream` will override the template specified with `tag`.
+
+By default, Docker uses the first 12 characters of the container ID to tag log messages.
```

Comment:
I've asked [here](https://github.com/docker/docker/pull/27707#discussion_r85618848) for this behavior to be changed back to what it was before your change.  Are you not planning to fix this?